package.json

```diff
@@ -7,128 +7,128 @@
     "npm": ">=3.8"
   },
   "dependencies": {
-    "babel-polyfill": "~6.26.0",
+    "@babel/polyfill": "~7.0.0-beta.44",
     "classnames": "~2.2.5",
     "font-awesome": "~4.7.0",
     "history": "~4.7.2",
-    "i18next": "~10.2.1",
-    "i18next-browser-languagedetector": "~2.1.0",
-    "lodash": "~4.17.4",
-    "moment": "^2.20.1",
+    "i18next": "~10.6.0",
+    "i18next-browser-languagedetector": "~2.1.1",
+    "lodash": "~4.17.5",
+    "moment": "^2.22.0",
     "moment-timezone": "^0.5.14",
     "normalize-url": "^1.9.1",
-    "normalize.css": "~7.0.0",
+    "normalize.css": "~8.0.0",
     "react": "~16.2.0",
     "react-dom": "~16.2.0",
     "react-fontawesome": "~1.6.1",
     "react-highlighter": "^0.4.2",
-    "react-i18next": "~7.1.1",
-    "react-onclickoutside": "~6.7.0",
-    "react-redux": "~5.0.3",
+    "react-i18next": "~7.5.1",
+    "react-onclickoutside": "~6.7.1",
+    "react-redux": "~5.0.7",
     "react-spinkit": "~3.0.0",
     "react-styled-flexboxgrid": "^2.2.0",
     "react-tooltip": "^3.4.0",
     "recompose": "^0.26.0",
     "redux": "~3.7.2",
-    "redux-actions": "~2.2.1",
+    "redux-actions": "~2.3.0",
     "redux-first-router": "^0.0.16-next",
     "redux-first-router-link": "^1.4.2",
     "redux-logger": "~3.0.6",
-    "redux-responsive": "^4.3.5",
+    "redux-responsive": "^4.3.6",
     "redux-thunk": "~2.2.0",
-    "styled-components": "^3.2.0",
+    "styled-components": "^3.2.5",
     "url-pattern": "^1.0.3",
-    "url-search-params-polyfill": "~2.0.1",
+    "url-search-params-polyfill": "~3.0.0",
```

Comment:
Do we still use the search params? I think I only used it for the weird pdf endpoint.